README.md

```diff
@@ -1,51 +1,39 @@
-[![GitHub Super-Linter](https://github.com/Calance-US/public-repository-template/workflows/Lint%20Code%20Base/badge.svg)](https://github.com/marketplace/actions/super-linter)
-
-
-## :black_nib: Project Title
-*Write about the project in a small paragraph*
+# :black_nib: Project Title
```

Comment:
Publish Allure Report Github Action

Reply:
@Ritik232  please replace `Project Title` with the title i have given in the above comment